web-wallet/src/routes/(app)/dashboard/receive/+page.svelte

```diff
@@ -0,0 +1,30 @@
+<svelte:options immutable={true} />
+
+<script>
+  import { onDestroy, onMount } from "svelte";
+  import { TransferContract } from "$lib/containers";
+  import { IconHeadingCard } from "$lib/containers/Cards";
+  import { contractDescriptors, updateOperation } from "$lib/contracts";
+  import { operationsStore } from "$lib/stores";
+  import { mdiArrowTopRight } from "@mdi/js";
+
+  onMount(() => {
+    updateOperation("receive");
+  })
+
+  onDestroy(() => {
+    updateOperation("receive");
```

Comment:
Why "receive" again?

Reply:
This is the operation for receiving tokens.